README.md

```diff
@@ -160,6 +160,9 @@ Chapter 11 - Function operators
 - Cohort 2: [Kevin Kent](https://twitter.com/kevin_m_kent) (SLIDES | [video](https://youtu.be/fD1QJB2pHik))
 - Cohort 3: [Anna-Leigh Brown](https://twitter.com/annaleighbrown2) (SLIDES) | [video](https://youtu.be/fW7Di01gLhw))
 
+Chapter 12 - OOP base types
+- Cohort 3: [Megn Stodel](https://twitter.com/MeganStodel) ([slides](https://github.com/MeganStodel/bookclub-Advanced_R/tree/cohort-3-base-types/Presentations/Week12/Cohort3))
```

Comment:
Just curious if you'd meant to sign your name like that :)

Reply:
Ugh...thanks! I've changed it now.